cirq/sim/state_vector_simulator.py

```diff
@@ -212,7 +212,7 @@ def state_vector(self):
                 |  6  |   1    |   1    |   0    |
                 |  7  |   1    |   1    |   1    |
         """
-        return self._final_simulator_state.state_vector
+        return self._final_simulator_state.state_vector.copy()
```

Comment:
Should we also have `test_final_state_vector_is_not_last_object` for this object?

Reply:
It is actually tested in sparse_vector_simulator_test (result.state_vector). But I've added a test to state_vector_simulator_test to test the copy here.